templates/master.yaml.tmpl

```diff
@@ -813,13 +811,36 @@ systemd:
       WantedBy=multi-user.target
   {{- end }}
 
+  {{ if eq .Provider "azure" -}}
```

Comment:
I see you removed the `{{if eq .Provider "aws" }}` elsewhere, why do we still need this unit for azure only?

Reply:
aws unit us above this one. I just duplicated it to not mess with conditions in the unit body.

Reply:
ok